src/main/java/de/internetsicherheit/brl/bloxberg/cache/gui/HistoricDataVisualizer.java

```diff
@@ -62,8 +62,8 @@ public void start(Stage primaryStage) throws IOException {
         // original: writer = new EthereumWriter(Path.of(OUTPUTDIRECTORY), filename);
         writer = new EthereumWriter(Path.of(OUTPUTDIRECTORY), filename);
         BlockAggregator dbs = initDataBlockSummerizer(filename);
-        TestCenter testCenter = new TestCenter(client, new File(OUTPUTDIRECTORY + "extractedData6.json"));
-        testCenter.generateJsonFile(0, 10000);
+        TestCenter testCenter = new TestCenter(client, new File(OUTPUTDIRECTORY + "transactionsFrom100kBlocks.json"));
+        testCenter.generateJsonFile(0, 100000);
```

Comment:
Why are these changes necessary in this PR?

Reply:
they are not absolutely necessary, but i like to change filenames frequently so I dont accidentally overwrite files.

Reply:
While I understand this, this means those filenames should not be hardcoded in the source code. So have them as arguments provided to your program or, at least, don't commit them, as this is creating noise in the git history.